File: Basic/Check Whether a Character is Vowel or Consonant/SolutionByRishabh.cpp
#include<iostream>
using namespace std;

void check(char ch)
{
	if(ch=='a' || ch=='e' || ch=='i' || ch=='o' || ch=='u')
		cout<<"Vowel";
	else
		cout<<"Constant";
}
int main()
{
	char ch;
	cin>>ch;

	check(ch);
}

File: Basic/Check Whether a Character is Vowel or Consonant/SolutionByRishabh.py
#Program to Check whether a character is vowel or consonent
n=input() #Taking th character from the user
#defining the function
def vowel(n):
    #Checking the character is in list of vowels or not
    if n in ["a","e","i","o","u","A","E","I","O","U"]:
        print("Output = Vowel")
    else:
        print("Output = Consonent")
#Calling the function
vowel(n)
